### ISSUE TITLE
Data Type Side Nav- 508 issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1490]
## Required Fix 

- [x] RF-1b 
- [x]  RF-7 

## Context 

See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-1b | None of the visual headings have html tags, they all appear to be divs. For example:  <div class="panel-header padding-top-3 padding-bottom-3 padding-left-2">Data Type |
| RF-7  | There is at least one form input field that does not have a label tag properly associated with it. This issue is a violation of Section 508 and of WCAG 2.0 Success Criterion 1.3.1 |

## Testing 
Tester: @aprematta

**Comment by junie-stigdon** [comment]
## Pull Request

#### RF-1b - Header Tags
- Gave "CAMPD: Clean Air Markets Division" an h1 tag
  - removed href
- Updated usa-footer__logo-heading to a h2 tag
- Updated .panel-header in global-overrides.scss to include header tags
- Gave h2 tags to the following:
  - Data Type (side panel header)
  - Filters (side panel header)
  - Custom Data Download (data preview header)
- Gave h3 tags to the following
  - Time Period (filter panel header)
  - Program (filter panel header)
  - State/Territory (filter panel header)
  - Facility (filter panel header)
  - Unit Type (filter panel header)
  - Unit Fuel Type (filter panel header)
  - Control Technology (filter panel header)
  - Data Preview (data preview header)
- Program filter sub headings:
  - h4 tag to Active and Retired
  - h5 tag to Annual and Ozone

#### RF-7 - Data Type Labels
- Created a Label for the Data Type dropdown
- Created a Label for the Data Sub Type dropdown

**Comment by aprematta** [comment]
Both issues pass testing, closing this ticket
